tests/Postman/addTestCartRule.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `salesrule` (`name`, `description`, `from_date`, `to_date`, `uses_per_customer`, `is_active`, `conditions_serialized`, `actions_serialized`, `stop_rules_processing`, `is_advanced`, `product_ids`, `sort_order`, `simple_action`, `discount_amount`, `discount_qty`, `discount_step`, `apply_to_shipping`, `times_used`, `is_rss`, `coupon_type`, `use_auto_generation`, `uses_per_coupon`, `simple_free_shipping`)
+VALUES
+	('App only discount','','2018-01-01','2030-01-01',0,1,'a:7:{s:4:\"type\";s:46:\"Magento\\SalesRule\\Model\\Rule\\Condition\\Combine\";s:9:\"attribute\";N;s:8:\"operator\";N;s:5:\"value\";s:1:\"1\";s:18:\"is_value_processed\";N;s:10:\"aggregator\";s:3:\"all\";s:10:\"conditions\";a:1:{i:0;a:5:{s:4:\"type\";s:48:\"Shopgate\\Base\\Model\\Rule\\Condition\\ShopgateOrder\";s:9:\"attribute\";s:17:\"is_shopgate_order\";s:8:\"operator\";s:2:\"==\";s:5:\"value\";s:1:\"1\";s:18:\"is_value_processed\";b:0;}}}','a:6:{s:4:\"type\";s:54:\"Magento\\SalesRule\\Model\\Rule\\Condition\\Product\\Combine\";s:9:\"attribute\";N;s:8:\"operator\";N;s:5:\"value\";s:1:\"1\";s:18:\"is_value_processed\";N;s:10:\"aggregator\";s:3:\"all\";}',0,1,NULL,0,'by_percent',10.0000,NULL,0,0,0,1,1,0,0,0);
```

Comment:
something is off with the conditions, I get an error when I import and go to the coupon page:
```
1 exception(s):
Exception #0 (InvalidArgumentException): Unable to unserialize value.

Exception #0 (InvalidArgumentException): Unable to unserialize value.
#0 /Users/konstantin/Development/shopgate/environments/mage2-vagrant/magento2ce/app/code/Magento/Rule/Model/AbstractModel.php(213): Magento\Framework\Serialize\Serializer\Json->unserialize('a:7:{s:4:"type"...')
#1 /Users/konstantin/Development/shopgate/environments/mage2-vagrant/magento2ce/app/code/Magento/SalesRule/Controller/Adminhtml/Promo/Quote/Edit.php(58): Magento\Rule\Model\AbstractModel->getConditions()
```